config/linux/ubuntu_install_timesketch.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+#
+# Script to install Timesketch on Ubuntu from the GIFT PPA.
+#
+# This file is generated by l2tdevtools update-dependencies.py any dependency
+# related changes should be made in dependencies.ini.
+
+# Exit on error.
+set -e
+
+# Install and configure Elasticsearch 7
+./config/linux/ubuntu_install_elasticsearch7.sh
+
+# Install and configure PostgreSQL
+sudo apt-get install -y postgresql python3-psycopg2
+
+sudo sed '$a local   all             timesketch                              md5' -i /etc/postgresql/10/main/pg_hba.conf
+
+sudo /etc/init.d/postgresql restart
+
+sudo -u postgres createuser -d -R -S timesketch
+sudo -u postgres psql -c "ALTER USER timesketch WITH PASSWORD 'tse2etest';"
+
+sudo -u postgres createdb -O timesketch timesketch
+
+# Install and configure Timesketch
+sudo add-apt-repository universe
+sudo apt-get update
```

Comment:
I'd do this at the beginning of the script

Reply:
The reason for having it here is to keep the TS specific stuff together.